docs/guide/installation-procedure.md

```diff
@@ -136,6 +136,59 @@ To work around this problem, such controllers should be launched as extern contr
 Before launching extern controllers, you should set the `WEBOTS_HOME` environment variable to point to `/snap/webots/current/usr/share/webots` and add `$WEBOTS_HOME/lib` to your `LD_LIBRARY_PATH` environment variable, so that your controllers will find the necessary shared libraries.
 The chapter entitled [running extern robot controllers](running-extern-robot-controllers.md) details how to run extern controllers, including with the snap version of Webots.
 
+#### Installing the Docker Image
+
+A [Docker](https://www.docker.com) image of Webots based on Ubuntu 18.04 is available on [dockerhub](https://hub.docker.com/r/cyberbotics/webots).
+
+This image can be used to use Webots in your continous-integration (CI) workflow without requiring any graphical user interface or to get a clean and sandboxed environment with Webots pre-installed including GPU accelerated graphical user interface.
+
+##### Install Docker
+
+Follow the [Docker installation instructions](https://docs.docker.com/engine/install/#server) to install docker.
+
+##### Run Webots in Docker without GUI
+
+The docker image comes with a X virtual framebuffer (Xvfb) already installed and configured so that you can run Webots in headless mode.
+
+To pull the image and start a docker container with it use the following command:
+```
+docker run -it cyberbotics/webots:latest /bin/bash
+```
+
+> **Note**: If you need a specific version of Webots and not the latest one, replace `latest` with the version you need (e.g. `R2020b-rev1`).
+
+After starting the docker container you can start Webots headlessly using xvfb:
+```
+xvfb-run webots --stdout --stderr --batch --mode=realtime /path/to/your/world/file
+```
+
+> **Note**: Since Webots runs in headless mode, the `--stdout` and `--stderr` arguments are used to redirect these streams from the Webots console to the console in which Webots was started, the `--batch` argument is there to make sure not blocking pop-up window is displayed and the `--mode=realtime` is used to make sure the simulation is not started in pause mode you might replace `realtime` by `fast`), finally don't forget to specify which simulation you want to run.
```

Comment:
```suggestion
> **Note**: Since Webots runs in headless mode, the `--stdout` and `--stderr` arguments are used to redirect these streams from the Webots console to the console in which Webots was started, the `--batch` argument is there to make sure no blocking pop-up window is displayed and the `--mode=realtime` is used to make sure the simulation is not started in pause mode (you might replace `realtime` by `fast`), finally don't forget to specify which simulation you want to run.
```